PlaygroundComponent.module.scss

```diff
@@ -0,0 +1,306 @@
+@import "./colors.module";
+
+.playground-content {
+    min-height: 95vh;
+    border-left: 2px solid var(--app-sidebar-border-color);
+
+    .header-title h1 {
+        text-align: center;
+        font-size: 48px;
+        margin-bottom: 40px;
+
+        @media (max-width: 992px) {
+            font-size: 32px;
+        }
+    }
+
+    .page-wrapper {
+        display: flex;
+        color: $black-color;
+
+        .playground {
+            display: flex;
+            justify-content: center;
+            flex-direction: row;
+            flex-wrap: wrap;
+            height: fit-content;
+            width: 100%;
+            margin-bottom: 160px;
+
+            @media (max-width: 992px) {
+                margin-bottom: unset;
+            }
+
+            &-page-wrapper {
+                display: flex;
+                width: 95%;
+
+                @media (max-width: 1078px) {
+                    flex-direction: column;
+                }
+            }
+        }
+    }
+}
+
+.playground-api-json {
+    display: flex;
+    flex-direction: column;
+    margin-right: 24px;
+
+    @media (max-width: 1078px) {
+        width: 100%;
+    }
+
+    .api-token {
+        display: flex;
+        flex-wrap: nowrap;
+        border-radius: 4px;
+        background-color: #f8fafb;
+        padding: 0 24px;
+        margin-bottom: 48px;
+        margin-top: 16px;
+
+        @media (max-width: 992px) and (max-width: 640px) {
+            flex-direction: column-reverse;
+        }
+
+        fieldset {
+            display: flex;
+            flex-wrap: nowrap;
+            align-items: center;
+            justify-content: flex-start;
+            width: 100%;
+            padding: 0 12px;
+            font-size: 14px;
+            line-height: 1.43;
+            border: none;
+            color: $black-color;
+            margin: 0 24px 0 0;
+
+            @media (max-width: 992px) and (max-width: 640px) {
```

Comment:
We can also specify the screen sizes with mobile screen states. this example could be replaced I think with
```
[data-state~="responsive.mobileL"],
[data-state~="responsive.mobileTablet"] {
 ....
}
```
This would be a responsive screen range of 425px to 1023px.